### PR TITLE
W-16312987-alt-text-trust-day-2.4

### PR DESCRIPTION
**Pull request by kevintroller**
-kt

# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/app-versioning.adoc [hunk 1]
```diff
@@ -35,7 +35,7 @@ In the *Config changes* tab on the application's *Settings* page. Runtime Fabric
 +
 The last ten configuration deployments are displayed in order of their respective configuration date. Each configuration in the history has a unique hash to identify it.
 
-image::app-versioning.png[]
+image::app-versioning.png["Configuration page for a runtime fabric application, displaying application file, version, deployment target, runtime version, and replicas"]
```
Comment (valkyrie69) [inline modules/ROOT/pages/app-versioning.adoc]:
```suggestion
image::app-versioning.png["Configuration page for a Runtime Fabric application, displaying the application file, version, deployment target, runtime version, and replicas"]
```

File: modules/ROOT/pages/app-versioning.adoc [hunk 2]
```diff
@@ -52,7 +52,7 @@ The *All changes* tab displays a history of versions created from the configurat
 . Once your changes are made, click *Apply Changes*. +
 At the top of the *Settings* page, a status bar appears.
 +
-image:app-versioning-applying-config.png[]
+image:app-versioning-applying-config.png["Applying configuration status for a Mule app in Runtime Manager UI"]
```
Comment (valkyrie69) [inline modules/ROOT/pages/app-versioning.adoc]:
```suggestion
image:app-versioning-applying-config.png["Applying configuration status for a Mule app in the Runtime Manager UI"]
```

File: modules/ROOT/pages/app-versioning.adoc [hunk 3]
```diff
@@ -74,17 +74,17 @@ This model means there is zero downtime while updates are applied to the applica
 [NOTE]
 Since updates are applied as rolling updates, there will be different application versions running at the same time until the update is completed, so make sure your applications are able to run multiple versions in parallel.
 +
-image::app-versioning-replica-status.png[]
+image::app-versioning-replica-status.png["Replicas Status for a Mule app in Runtime Manager UI"]
```
Comment (valkyrie69) [inline modules/ROOT/pages/app-versioning.adoc]:
```suggestion
image::app-versioning-replica-status.png["Status of replicas for a Mule app in the Runtime Manager UI"]
```

File: modules/ROOT/pages/app-versioning.adoc [hunk 4]
```diff
@@ -74,17 +74,17 @@ This model means there is zero downtime while updates are applied to the applica
...
 +
 Once the configuration is successfully applied and deployed, a message appears at the top of the page.
 +
-image::app-versioning-config-applied-success.png[]
+image::app-versioning-config-applied-success.png["Configuration applied successfully for a Mule app in Runtime Manager UI"]
```
Comment (valkyrie69) [inline modules/ROOT/pages/app-versioning.adoc]:
```suggestion
image::app-versioning-config-applied-success.png["Configuration applied successfully status for a Mule app in Runtime Manager UI"]
```

File: modules/ROOT/pages/app-versioning.adoc [hunk 5]
```diff
@@ -74,17 +74,17 @@ This model means there is zero downtime while updates are applied to the applica
...
 +
 The application status, configuration identifier, date, and other information is updated in the status box.  The new configuration is displayed at the top of the *Config changes* list, along with a *Last successful* label to indicate that it's the last configuration successfully deployed.
 +
 If the updates to the configuration fail to deploy, you get an `Desired configuration wasn't applied` message, and the *Configuration status* displays an `Update failed` label. The configuration is displayed in the *Config changes* history without a label.
 +
-image::app-versioning-update-failed.png[]
+image::app-versioning-update-failed.png["Configuration failed to apply for a Mule app in Runtime Manager UI"]
```
Comment (valkyrie69) [inline modules/ROOT/pages/app-versioning.adoc]:
```suggestion
image::app-versioning-update-failed.png["Configuration failed to apply status for a Mule app in the Runtime Manager UI"]
```

File: modules/ROOT/pages/app-versioning.adoc [hunk 6]
```diff
@@ -74,17 +74,17 @@ This model means there is zero downtime while updates are applied to the applica
 [NOTE]
 Since updates are applied as rolling updates, there will be different application versions running at the same time until the update is completed, so make sure your applications are able to run multiple versions in parallel.
 +
-image::app-versioning-replica-status.png[]
+image::app-versioning-replica-status.png["Status of replicas for a Mule app in the Runtime Manager UI"]
```
Comment (dmerlob) [inline modules/ROOT/pages/app-versioning.adoc]:
```suggestion
image::app-versioning-replica-status.png["Replicas status for a Mule app in the Runtime Manager UI"]
```
Or you can use "Status of replicas" in all other PR entries to match the content.

**Review by dmerlob — DISMISSED** [review]

**Review by dmerlob — APPROVED** [review]

**Pull request merged**